scapy/layers/tls/__init__.py

```diff
@@ -1,24 +1,121 @@
 ## This file is part of Scapy
-## See http://www.secdev.org/projects/scapy for more informations
-## Copyright (C) Arnaud Ebalard, Maxence Tury
+## Copyright (C) 2007, 2008, 2009 Arnaud Ebalard <arno@natisbad.com>
+##                     2015, 2016 Maxence Tury <maxence.tury@ssi.gouv.fr>
 ## This program is published under a GPLv2 license
 
 """
 Tools for handling TLS sessions and digital certificates.
+
+Prerequisites:
+
+    - You may need to 'pip install cryptography' for the module to be loaded.
+
+
+Main features:
+
+    - X.509 certificates parsing/building.
+
+    - RSA & ECDSA keys sign/verify methods.
+
+    - TLS records and sublayers (handshake...) parsing/building. Works with
+      versions SSLv3, TLS 1.0, 1.1 and 1.2. This may be enhanced by a TLS
+      context. For instance, if scapy reads a ServerHello with version TLS 1.2
+      and a cipher suite using AES, it will assume the presence of IVs
+      prepending the data. See test/tls.uts for real examples.
+
+    - TLS encryption/decryption capabilities with the usual ciphersuites. Once
+      again, the TLS context enables scapy to transparently send/receive
+      protected data if it learnt the session secrets. Note that if scapy acts
+      as one side of the handshake (e.g. reads all server-related packets and
+      builds all client-related packets), it will indeed compute the session
+      secrets.
+
+    - TLS client & server basic automatons, provided for testing and tweaking
+      purposes. These make for a very primitive TLS stack.
+
+    - Additionally, a basic test PKI (key + certificate for a CA, a client and
+      a server) is provided in tls/examples/pki_test.
+
+
+Unit tests:
+
+    - Various cryptography checks.
+
+    - Reading a TLS handshake between a Firefox client and a GitHub server.
```

Comment:
In the secdev team, they love calling "secdev.org" rather than any other servers :)